_data/projects/AutoSort.yml

```
@@ -0,0 +1,10 @@
+name: AutoSort
+desc: Sort by a string or by default attributeA brief description of the project
+site: https://github.com/ming-tsai/NetCore.AutoSort
+tags:
+  - c#
+  - netcore
+  - dotnet3-1
```

Let's avoid focusing on specific versions of frameworks wherever possible:

```suggestion
  - netcore
```